tee-worker/litentry/core/assertion-build/src/a1.rs

```diff
@@ -25,9 +25,9 @@ use itp_stf_primitives::types::ShardIdentifier;
 use itp_types::AccountId;
 use lc_credentials::Credential;
 use lc_stf_task_sender::MaxIdentityLength;
-use litentry_primitives::{Assertion, Identity, ParentchainBlockNumber};
+use litentry_primitives::{Identity, ParentchainBlockNumber};
 use log::*;
-use parachain_core_primitives::VCMPError;
+use parachain_core_primitives::{Assertion, VCMPError};
```

Comment:
We shouldn't have both `litentry_primitives` and `parachain_core_primitives`, we should only depend on `litentry_primitives`.

If you need `Assertion`, you should re-import it in the `litentry_primitives`: `pub use parentchain_primitives::Assertion` and use it here: `use litentry_primitives::Assertion`

Same applies to other assertion-build files